templates/zerver/emails/sponsorship_approved_community_plan.subject.txt

```diff
@@ -0,0 +1 @@
+{% trans %}Community plan sponsorship approved for {{ billing_entity }}!{% endtrans %}
```

Comment:
@alya do we want Zulip in the subject line? I think we don't necessarily need it given that it's in the sender name.

Reply:
Yeah, I think it's fine without.